.rustfmt.toml

```
@@ -0,0 +1,15 @@
+condense_wildcard_suffixes = true
+enum_discrim_align_threshold = 20
+struct_field_align_threshold = 20
+group_imports = "StdExternalCrate"
+imports_indent = "Block"
+fn_single_line = true
+format_strings = true
+imports_granularity = "Item"
```

About the comment on import style, I think `Module` would give the expected behavior.
```suggestion
imports_granularity = "Module"
```